### ISSUE TITLE
problem with glue

### ISSUE THREAD

**Issue by s-bravo** [datawookie/emayili#39]
Hi, Im working with twitteR, tidyverse and Emayili. 
When i run the script just with tidyverse and Emayili, it send perfectly the email. But when I run the script with twitteR, it give me this notification.

```
"Error: glue cannot interpolate functions into strings.
* object 'name' is a function."
```
here an example of my scrip.

```
library(twitteR);library(tidyverse);library(emayili)

consumerkey <- "*************"
  consumersecret <- "*************"
  accesstoken <- "*************"
  accessTokenSecret <- "*************"
  setup_twitter_oauth(consumerkey, consumersecret, accesstoken, accessTokenSecret)

 SEARCH1 = twitteR::searchTwitter('search', n = 1000, resultType="recent")  %>%
    strip_retweets()

  SEARCH1 = twitteR::twListToDF(SEARCH1)

teste <- envelope() %>%
   from("mail@mail.com") %>%
   to("mail@mail.com") %>%
   subject("text tet") %>%
   html('<p>hi</p>')
 
 smtp <- server(host = "***",
                         port = 25,
                         username = "***",
                         password = "***")
 

 smtp(teste, verbose = TRUE)
)

```

any idea about what happening? 
THANKS!

**Comment by datawookie** [comment]
Hi Sebastian,

Thanks for bringing this to my attention.

It turns out that `{twitteR}` was a red herring and you could reproduce the problem with the code below.

```r
library(twitteR)
library(dplyr)
library(emayili)

teste <- envelope() %>%
  from("mail@mail.com") %>%
  to("mail@mail.com") %>%
  subject("text test") %>%
  html('<p>hi</p>')

smtp <- server(host = SMTP_SERVER, port = 8025)

smtp(teste, verbose = TRUE)
```

The problem was that `glue()` was trying to find the `name` variable in the global namespace and found the one from `{twitteR}`, which is a function (not a variable).

Should be fixed by #40.

Best regards,
Andrew.

**Comment by s-bravo** [comment]
Perfect!! I will test! 
Thanks
